content/e20co/drone-marathon/index.md

```diff
@@ -78,7 +78,19 @@ Will result in:
 
 # Secrets
 
-Not Implemented.
+The Marathon plugin supports reading credentials from the Drone secret store. This is highly recommended instead of storing credentials in the pipeline configuration in plain text.
+
+```diff
+pipeline:
+  marathon:
+    image: e20co/drone-marathon:0.5
+    server: http://marathon.mesos:8080
+    values:
+-     DATABASE_PASSWORD: SomeSecretPassword
++     DATABASE_PASSWORD: ${DATABASE_PASSWORD}
```

Comment:
I'm not sure I understand safe vs unsafe here. Drone has a fixed list of environment variables that you can see at http://readme.drone.io/usage/environment-interpolation/

The kubernetes plugin, for example, uses `os.ExpandEnv(...)` to interpolate all environment variables into their configuration file. What is the safety issue in doing so? All variables are prefixed with `DRONE_` and most secrets require `--image` to filter which plugins they are injected into